usagov/vote-gov-drupal
# VOTE-2082/2083: High contrast styling on basic and state pages

## Jira ticket

[VOTE-2082](https://cm-jira.usa.gov/browse/VOTE-2082)
[VOTE-2083](https://cm-jira.usa.gov/browse/VOTE-2083)

## Description

Insert a brief summary of the changes included in this PR and any additional information or context which may help the reviewer.

## Deployment and testing

### Post-deploy steps

1. Insert any required steps to take before beginning test.

### QA/Testing instructions

1. lando retune
2. npm run build in theme
3. Navigate to a state page and a basic page
4. Toggle the high contrast button and verify styles
5. Toggle back to default, verify styles

## Checklist for the Developer




## Checklist for the Peer Reviewers
